model/trace/messaging.yaml

```diff
@@ -82,15 +82,11 @@ groups:
       - ref: messaging.message.body.size
       - ref: server.address
       - ref: network.peer.address
-        tag: connection-level
```

Comment:
Are these changes to messaging semconv intended to be part of this PR?

Reply:
Same question applies to the changes in `messaging-common.yaml`

Reply:
this is a dead code in messaging semconv which was copied over from DB. 

I'd rather keep it, but can remove this change if you feel strong about it.